#!/usr/bin/env python3
PACKAGE = "follow_lane_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("thresh", int_t, 0, "Thresh Value", 180, 0, 255)
gen.add("speed", double_t, 0, "Speed Value", 1.0, 0.0, 4.0)
gen.add("enable_drive", bool_t, 0, "Enable", False)

gen.add("percent_white_max", double_t, 0, "Percent Value", 3, 0, 100)
gen.add("percent_white_min", double_t, 0, "Percent Value", 2, 0, 100)



exit(gen.generate(PACKAGE, "follow_lane", "FollowLaneLuke"))
